hoot-core/src/main/cpp/hoot/core/io/ChangesetDeriver.cpp

```diff
@@ -140,7 +140,7 @@ Change ChangesetDeriver::_nextChange()
       _fromE = _from->readNextElement();
       if (_fromE)
       {
-        LOG_TRACE("Next 'from' element: " << _toE->getElementId());
+        LOG_TRACE("Next 'from' element: " << _fromE->getElementId());
```

Comment:
yeah, i'm sure that was not helping me debug at all over the last couple weeks :-(